src/System.Private.ServiceModel/src/project.json

```diff
@@ -21,7 +21,7 @@
     "System.Net.Primitives": "4.0.10",
     "System.Net.WebHeaderCollection": "4.0.0",
     "System.Net.WebSockets": "4.0.0",
-    "System.Net.WebSockets.Client": "4.0.0",
+    "System.Net.WebSockets.Client": "4.0.1-beta-24322-03",
```

Comment:
Just to point out, we now officially cannot release a stable version until `System.Net.WebSockets.Client` does.